cmd/data.go

```diff
@@ -44,10 +44,10 @@ type DataCmd struct {
 	targetProfile   string
 	sessionJSON     string
 	filePrefix      string // TODO: move filePrefix to global flags
-	writeLimit      int64
+	WriteLimit      int64
 	dryRun          bool
 	logLevel        string
-	skipForeignKeys bool
+	SkipForeignKeys bool
```

Comment:
Why are these capitalised? Lets keep them the same as before to avoid confusion.
(I know the earlier convention used might be wrong but changing case for only some variables decreases readability)

Reply:
Made them capitalised because we want to refer them in web.go file